augmentedreality/Common/source/LocationDataSource.cpp

```diff
@@ -40,7 +40,7 @@ using namespace Esri::ArcGISRuntime::Toolkit;
   are emitted every time the values change.
 
   Most of the time, it's not necessary to create an object of this class. A default LocationDataSource
-  is created automatically by \l ArcGISArSceneView when this is necessary. This class is public to
+  is created automatically by \l ArcGISArView when this is necessary. This class is public to
```

Comment:
Not sure if this is the correct class. Had a look on it and it seems that does what the text is saying. 
`ArcGISSceneView` could not be found. Might have been deleted?

Reply:
❓  I'm not too familiar with AR to say what is correct here. @GuillaumeBelz thoughts?

Reply:
The type should be `ArcGISArView` - https://github.com/Esri/arcgis-runtime-toolkit-qt/blob/main/augmentedreality/QmlApi/qml/ArcGISArView.qml